src/Microsoft.Dnx.Tooling/Program.cs

```diff
@@ -80,7 +80,28 @@ public int Main(string[] args)
             WrapConsoleCommand.Register(app, reportsFactory);
             FeedsConsoleCommand.Register(app, reportsFactory);
 
-            return app.Execute(args);
+            try
+            {
+                return app.Execute(args);
+            }
+            catch (CommandParsingException ex)
+            {
+                AnsiConsole.GetError(_runtimeEnv.OperatingSystem == "Windows").WriteLine(ex.Message.Red().Bold());
+                ex.Command.ShowHelp();
+                return 1;
```

Comment:
I've seen some discrepancies with error codes. In the application host I see we return -1 on errors. Elsewhere I've seen 1. Is there a reason to use one or the other? @davidfowl @anurse